python/PresentationBuilder/slidesets/CoverSet.py

```diff
@@ -88,3 +109,11 @@ def contents(self):
     for i in range(len(output)):
       contents_name = self.name() + '-contents-' + str(i)
       self._slides[contents_name].markdown += output[i]
+
+  ##
+  # Return true if the slide set is active for contents
+  def _isSetActive(self, slide):
+    parent = slide.getParam('_parent')
+    if parent.name() in self.inactive or (self.active != '' and parent.name() not in self.active):
```

Comment:
Yes, this comes from an input file. So, the user can do what they want. In
that case, 'inactive' wins.

On Fri, Jan 30, 2015 at 9:31 AM, John W. Peterson <notifications@github.com>
wrote:

> In python/PresentationBuilder/slidesets/CoverSet.py
> <https://github.com/idaholab/moose/pull/4625#discussion_r23853100>:
>
> > @@ -88,3 +109,11 @@ def contents(self):
> >      for i in range(len(output)):
> >        contents_name = self.name() + '-contents-' + str(i)
> >        self._slides[contents_name].markdown += output[i]
> > +
> > +  ##
> > +  # Return true if the slide set is active for contents
> > +  def _isSetActive(self, slide):
> > +    parent = slide.getParam('_parent')
> > +    if parent.name() in self.inactive or (self.active != '' and parent.name() not in self.active):
>
> Can a name be in both self.inactive *and* self.active?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/moose/pull/4625/files#r23853100>.
>